test/integration/forms/form.js

```diff
@@ -389,7 +448,8 @@ context('Patient Action Form', function() {
       .wait('@routeLatestFormResponseByPatient')
       .itsUrl()
       .its('search')
-      .should('contain', 'filter[form]=11111');
+      .should('contain', 'filter[form]=11111')
+      .should('not.contain', 'filter[flow]');
```

Comment:
we should go ahead and add `fx.data.relationships.flow = { data: { id: '1' } };` to the `routeAction`.  Even if the action has a flow it should not show up in this test.

Reply:
Updated in last push.